hyperjumptech/monika
# Refactor symonClient initiate function

# Monika Pull Request (PR)

## What feature/issue does this PR add
1. to fix #1174 
2. refactoring event emitter on symonClient initiate function

## How did you implement / how did you fix it
1. refactor callback function inside notification.willSend event emitter
2. set alertId to be sent from an event emit

## How to test
1. run monika in symon mode
2. make sure the notification is sent
